### DISCUSSION TITLE
Twitter API No longer free.

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#2058] [General]
I don't know if there is somehow around this. On feb 9th elon will decide to monetize it.
https://twitter.com/TwitterDev/status/1621026986784337922

**Comment by Harmon758** [comment]
More accurately:
> Starting February 9, we will no longer support free access to the Twitter API, both v2 and v1.1. A paid basic tier will be available instead
> 
> We’ll be back with more details on what you can expect next week.

There is no further information about this right now, so there's nothing to be done about it yet.

Personally, I am very opposed to this change though.

**Comment by SakiSakiSakiSakiSaki** [comment]
Just when I started working on my program using Tweepy too. 

Unfortunate.

**Comment by prahladyeri** [comment]
Today is 9th February. What's the update on this?

**Reply by Harmon758** [reply]
https://twitter.com/TwitterDev/status/1623467615539859456

**Comment by Keinta15** [comment]
Looks like it's been delayed further
https://twitter.com/TwitterDev/status/1625234161010343941

This time they don't specify a date

**Reply by prahladyeri** [reply]
> Looks like it's been delayed further https://twitter.com/TwitterDev/status/1625234161010343941
> 
> This time they don't specify a date

@Keinta15 But something definitely seems to have changed. I remember that earlier, the [Twitter Developer Portal](https://developer.twitter.com/en/docs/twitter-api) used to provide us API Access Token Keys to access the API through code. But now, that interface has entirely changed. It is asking for a sign-up with detailed form submission for "Hobbyist", "Academic Researcher", etc. Is this API still really free to use?

**Reply by Harmon758** [reply]
I don't see any changes to the Developer Portal.

**Reply by prahladyeri** [reply]
OK. I might have missed something as I'm seeing it after a very long time! It's just that my old API token isn't working and I don't remember how exactly on the developer portal I had gotten it.
